newToken.tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "compilerOptions": {
```

Comment:
There appears to already be a tsconfig in the project. Do we need another one? Or is this doing something the other isn't?

Reply:
The normal tsconfig is sued for the `build.ts` in `scripts/` which is not the same. The goal is to combine all, but for this we need to port colors.

Ideally we would only have one, but I tried to get this bundling start with the least changes possible.

Reply:
okay, I think you could just extend from that original version then? and add only the delta between the two to this new one? 

```json
{
  "extends": "../tsconfig.json",
  "compilerOptions": {
    ...new stuff only for your build file
  }
}
```

Reply:
Yes, we can do that but most props are overwritten. When colors are moved we can hopefully remove the old one in any case

Reply:
🤔 okay, to avoid any confusion for other contributors in the interim, can you rename the file to be `v2-tokens.tsconfig.json` or similar instead of `new.tsconfig.json`?